app/controllers/admin/runs_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class Admin::RunsController < Admin::BaseController
   def index
-    authorize :runs, :index?
+    authorize :run, :index?
```

Comment:
Yes, the update bumped pundit form 0.3.0 to 1.0.something and I guess there has been a change in the authorize method.